vagrant/ansible/roles/ctdb.setup/tasks/main.yml

```diff
@@ -74,6 +77,7 @@
     gluster_features_ctdb_nodes: "{{ ctdb_network_private_interfaces | join(',') }}"
     gluster_features_ctdb_publicaddr: "{{ ctdb_network_public_interfaces | map('regex_replace','(^.*$)','\\1/' + ctdb_network_public_interface_subnet_mask + ' ' + ctdb_network_public_interface_name) | list |join (',') }}"
     gluster_features_ctdb_volume: "{{ ctdb_glusterfs_volume }}"
+    ctdb_glusterfs_volume: "{{ ctdb_cluster_volume }}"
```

Comment:
Why can't we directly assign _ctdb_cluster_volume_ to _gluster_features_ctdb_volume_ ?

Reply:
done.